amethyst_rendy/src/bundle.rs

```diff
@@ -23,18 +23,64 @@ use amethyst_core::{
 use amethyst_error::{format_err, Error};
 use std::collections::HashMap;
 
+/// A bundle of systems used for rendering using `Rendy` render graph.
+///
+/// It's highly recommended to use `PluggableRenderingBundle` as it is easier to construct and manage
+/// at the loss of advanced manual handling.
```

Comment:
Had to re-read this sentence 2-3 times.
Can it be replaced by: "It is highly recommended to use `PluggableRenderingBundle`, as it is easier to construct and manage.

However, at the opposite of `PluggableRenderingBundle`, this (`RenderingBundle`) allows for fully customizable ren......"